Dockerfile.authnode

```diff
@@ -1,8 +1,9 @@
-FROM python:3.8.5-alpine3.12
+FROM python:3.9.2-alpine3.13
 
 RUN apk add gcc musl-dev python3-dev libffi-dev openssl-dev linux-headers make g++ 
 RUN apk add py3-grpcio>1.29.1 py3-protobuf
 RUN apk upgrade
+RUN apk add rust cargo
```

Comment:
how did we end up depending on rust? And not just a rust binary but the rust build toolchain?

Reply:
Python cryptography

Reply:
why does python cryptography require a rust toolchain? I get that it might be implemented in rust but shouldn’t we be able to fetch it as a compiled artifact?

Reply:
Afaict this is an issue on alpine. There is not a binary version of wheel and it compiles from source